resources/views/InventoryList/InventoryList.tsx

```diff
@@ -1,10 +1,12 @@
+
 import {Button, Col, Form, Input, InputNumber, Modal, Row, Table, Typography} from 'antd'
 import React, {useEffect, useState} from 'react'
 import {StoreType, DispatchArgumentType} from '@types'
 import {connect} from 'react-redux'
 import styled from 'styled-components'
 import {InventoryItemStateType} from 'models/inventory'
 import {filterInventory, getInventory} from 'services/inventory'
+import DeleteButton from '../../components/DeleteButton/DeleteButton'
```

Comment:
make sure you export from component index file then you can just do 
`import DeleteButton from 'components'`